### PR TITLE
Fix mint test failure in s3 gateway mode

### PR DESCRIPTION
**Pull request by prakashsvmx**
Fix mint test failure in s3 gateway mode

tested with 
- S3 Gateway
- MinIO server

### CODE REVIEW

**Comment by vadmeste** [comment]
@prakashsvmx do you know if package-lock.json is really needed to be committed ?

**Comment by prakashsvmx** [comment]
Yes @vadmeste . looks like it has been missed earlier.

File: src/main/minio.js [hunk 1]
```diff
@@ -641,12 +640,12 @@ export class Client {
         if(err.code === "AuthorizationHeaderMalformed" && err.region !== ""){
           // Retry with region returned as part of error
           this.makeRequest({method, bucketName, headers}, payload, [200], err.region, false, cb)
+        }else {
```
Comment (harshavardhana) [inline src/main/minio.js:643]:
```suggestion
        } else {
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: src/main/minio.js (hunk 1 above)
Comment (prakashsvmx) [reply src/main/minio.js]: Thank you. i have updated.

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**